redis/commands.py

```diff
@@ -1649,6 +1649,8 @@ def xadd(self, name, fields, id='*', maxlen=None, approximate=True,
         maxlen: truncate old stream members beyond this size
         approximate: actual stream length may be slightly more than maxlen
         nomkstream: When set to true, do not make a stream
+        minid: the minimum id in the stream to query
+        limit: specifies the maximum number of entries to retrieve
         """
         pieces = []
         if maxlen is not None:
```

Comment:
Maxlen and minid are mutually exclusive. This would allow us to specify both in the command.
https://redis.io/commands/XADD

Reply:
I tested it in L2449. 
They need to specify a threshold, so I can't merge them into one variable anyway,.